test/boost/view_build_test.cc

```diff
@@ -426,30 +433,31 @@ SEASTAR_TEST_CASE(test_view_update_generator) {
         auto sst = t->make_streaming_staging_sstable();
         sstables::sstable_writer_config sst_cfg = test_sstables_manager.configure_writer();
         auto& pc = service::get_local_streaming_write_priority();
+
         sst->write_components(flat_mutation_reader_from_mutations({m}), 1ul, s, sst_cfg, {}, pc).get();
         sst->open_data().get();
         t->add_sstable_and_update_cache(sst).get();
         view_update_generator.register_staging_sstable(sst, t).get();
 
-        eventually([&] {
-            auto msg = e.execute_cql("SELECT * FROM t WHERE p = 'a'").get0();
+        eventually([&, key1, key2] {
```

Comment:
Isn't it superfluous to capture key1, key2 by value in addition to capturing everything by reference? `eventually()` is synchronous anyway. Still, it's nonblocking, in a test, and I don't really care, so I'll commit this series as is. I just wanted to get the feeling of pride one gets when finding a nitpick. We also have `format()` in seastar namespace by the way, so no need to use `fmt::format`.

Reply:
you guy are just messing with me, aren't you ?

Reply:
I can't resist, since you mentioned the adrenaline spikes. I'm actually in the middle of merging this one, so please don't force-push a v3.